### ISSUE TITLE
Doctrine: Generation of ORM indexes automatically

### ISSUE THREAD

**Issue by azatyan** [Haehnchen/idea-php-annotation-plugin#35]
Add the way to generate ORM indexes **with single click**.
Update indexes when clicked on generated indexes entity
```php
@ORM\Index(name="index_name", columns={"field_name"})
@ORM\Index(name="second_index_name", columns={"other_field_name"})
// other indexes ...
```

**Comment by Haehnchen** [comment]
good idea. iam searching for the right sytax in differnt metadata all the time :)

**Comment by azatyan** [comment]
annotations are in official docs => http://docs.doctrine-project.org/projects/doctrine-orm/en/latest/reference/annotations-reference.html#annref-index  (use with prefixed ORM\Index)
and how can i help implement without Java skills ?))

**Comment by Haehnchen** [comment]
https://github.com/Haehnchen/idea-php-shopware-plugin/issues/30

**Comment by azatyan** [comment]
(y)

**Comment by azatyan** [comment]
@Haehnchen looks now you can implement it?
